website/docs/reference/resource-configs/target_schema.md

```diff
@@ -53,38 +53,14 @@ snapshots:
 
 </File>
 
-### Use a target-aware schema
-Use the [`{{ target }}` variable](/reference/dbt-jinja-functions/target) to change which schema a snapshot <Term id="table" /> is built in.
-
-Note: consider whether this use-case is right for you, as downstream `refs` will select from the `dev` version of a snapshot, which can make it hard to validate models that depend on snapshots (see above [FAQ](#faqs))
-
-<File name='dbt_project.yml'>
+<VersionBlock lastVersion="1.8" >
 
-```yml
-snapshots:
-  +target_schema: "{% if target.name == 'prod' %}snapshots{% else %}{{ target.schema }}{% endif %}"
-
-```
+### Use a target-aware schema
```

Comment:
I think we can get rid of this section entirely and just have the "Use the same schema-naming behavior as models" one?

Reply:
okee dokee! i'll get rid of this section completely